src/Illuminate/Redis/Connections/PredisConnection.php

```diff
@@ -52,20 +50,4 @@ public function createSubscription($channels, Closure $callback, $method = 'subs
 
         unset($loop);
     }
-
-    /**
-     * Flush the selected Redis database.
-     *
-     * @return void
-     */
-    public function flushdb()
-    {
-        if (! $this->client->getConnection() instanceof ClusterInterface) {
-            return $this->command('flushdb');
-        }
-
-        foreach ($this->getConnection() as $node) {
-            $node->executeCommand(new ServerFlushDatabase);
-        }
-    }
```

Comment:
Removing this here is a breaking change.

Reply:
why? The flushdb will go to the client, I moved it as recommended by @tillkruss

Reply:
There's no longer a flushdb method on PredisConnection. Any app calling it will break.

Reply:
https://github.com/laravel/framework/blob/45995483913655b7061cdec58421539096afb7b9/src/Illuminate/Redis/Connections/Connection.php#L214

Won't it get called here? I believe I tested it and there was no issues. Maybe I am misunderstanding something

Reply:
Yes but the behaviour won't be applied anymore. It'll just propagate to `command`. Maybe I'm missing something as well but I'm just seeing removed functionality 😅

Reply:
I meant to only extract part of the method @bernardwiesner. This must stay:

```php
/**
      * Flush the selected Redis database.
      *
      * @return void
      */
     public function flushdb()
     {
         return $this->command('flushdb');
     }
```